### PR TITLE
Created meeting notes for August 11th 2016 meeting

### PR DESCRIPTION
**Pull request by nebrius**

### CODE REVIEW

File: meetings/2016-08-11.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+(doc to move to nodejs/node repository after meeting)
+
+# Node Foundation TSC Meeting 2016-08-11
+
+## Links
+
+* **Audio Recording**: TBP
```
Comment (williamkapke) [inline meetings/2016-08-11.md]: https://www.youtube.com/watch?v=DKXD3anBh94

File: meetings/2016-08-11.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* **GitHub Issue**:
```
Comment (williamkapke) [inline meetings/2016-08-11.md]: https://github.com/nodejs/TSC/issues/127

File: meetings/2016-08-11.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+(doc to move to nodejs/node repository after meeting)
```
Comment (williamkapke) [inline meetings/2016-08-11.md]: remove this before merge plz

**Comment by mhdawson** [comment]
LGTM with william's additions.  Minor nit you can consider, we might want to add github handles for people as otherwise it may not be clear who people are from just their first name.

**Comment by nebrius** [comment]
Fixed nits, and I also realized I forgot to add the "Present" section. I think I added everyone, but please double check.

**Comment by williamkapke** [comment]
LGTM

**Comment by nebrius** [comment]
@mhdawson, does this look good to merge?

**Pull request merged**